Update zed to v0.30.0 and prep v0.25.0 (#1834)
brimdata/brim: CHANGELOG.md

## v0.25.0

> As you can see below, there's been many changes since the last Brim GA release!  Highlights include:

Reviewer: ```suggestion
As you can see below, there've been many changes since the last Brim GA release!  Highlights include:
```

* The storage used by Brim to hold your logs is now a Zed lake. Though the
  introduction of Zed lakes causes no immediate change to your favorite Brim
  workflows, they unlock powerful new functionality that will be revealed in
  Brim going forward, including Git-like branching. See the
  [Zed lake README](https://github.com/brimdata/zed/blob/main/docs/lake/README.md)
  for details.
* Enhancements have been made to the Zed language to unify search and
  expression syntax, introduce new operators and functions for data
  exploration and shaping, and more! Review the
  [Zed language docs](https://github.com/brimdata/zed/blob/main/docs/language/README.md)
  for details.

> * pcap-processing functionality is now handled by a separate new component

Reviewer: ```suggestion
* pcap processing is now handled by a separate, new component
```

  called Brimcap. Your favorite pcap workflows in Brim have not changed, but
  Brimcap also opens up new flexible custom configurations and can be used as

>   a separate standalone tool. For more info, check out the

Reviewer: ```suggestion
  a standalone tool. For more info, check out the
```

  Brimcap [README](https://github.com/brimdata/brimcap/blob/main/README.md)
  and [wiki](https://github.com/brimdata/brimcap/wiki).

> Among the many detailed changes listed further below, there's a few big ones in

Reviewer: ```suggestion
Among the many detailed changes listed below, there're a few big ones in
```
